container/worker/run_openqa_worker.sh

```diff
@@ -24,4 +24,14 @@ fi
 qemu-system-x86_64 -S &
 kill $!
 
+if [[ -e /var/lib/openqa/share/tests ]] ; then
+  pushd /var/lib/openqa/share/tests
+    for dir in $(ls) ; do
```

Comment:
`$(ls)` should be equivalent to `*`, isn't it?

Reply:
`*` would be better because it doesn't break when using white spaces. However, I'd prefer using `find`.